testbed/testbed/child_process.go

```diff
@@ -130,7 +153,7 @@ type ResourceConsumption struct {
 // the process to.
 // cmd is the executable to run.
 // cmdArgs is the command line arguments to pass to the process.
-func (cp *childProcess) start(params startParams) error {
+func (cp *ChildProcess) Start(params StartParams) (string, error) {
```

Comment:
Unclear what does the first `string` return value do.

Reply:
Added documentation